KinSDK/KinSDK/Core/blockchain/types/WhitelistPayload.swift

```diff
@@ -1,22 +1,22 @@
 //
-// WhitelistEnvelope.swift
-// KinSDK
+//  WhitelistPayload.swift
+//  KinSDK
 //
-// Created by Kin Foundation.
-// Copyright © 2018 Kin Foundation. All rights reserved.
+//  Created by Corey Werner on 01/07/2019.
+//  Copyright © 2019 Kin Foundation. All rights reserved.
 //
 
 import Foundation
 
 /**
- `WhitelistEnvelope` wraps a `TransactionEnvelope` and the `Network.Id`.
+ `WhitelistPayload` wraps a `Transaction.Envelope` and the `Network.Id`.
  */
-public struct WhitelistEnvelope {
-    
+public struct WhitelistPayload {
```

Comment:
why `WhitelistPayload` not available directly from the Transaction/TransactionEnvelope as a property?

Reply:
like this? https://github.com/kinecosystem/kin-sdk-ios/pull/49/files#diff-6c97de163866eac3371e8e5219c3d676R44